docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+services:
+  server:
+    image: node:14
+    working_dir: /server
+    ports: 
+      - 3000:3000
+    environment: 
+      - PGUSER=my-app
+      - PGHOST=db
+      - PGPASSWORD=my-app-pass
+      - PGDATABASE=my-app-db
+      - PGPORT=5432
+    depends_on:
+      - db
+    volumes:
+      - type: bind
+        source: ./server
+        target: /server
+    command: bash -c "npm install && npm start"
+
+  db:
+    image: postgres:latest
+    ports:
+      - 35432:5432
```

Comment:
A postgres has its port already set to 5432 by default. Therefore, you can remove lines 24 and 25